pkg/hvs/services/hosttrust/saml_report.go

```diff
@@ -119,17 +120,20 @@ func getHardwareFeaturesMap(features model.HardwareFeatures) map[string]string {
 
 	hwFeaturesMap := make(map[string]string)
 	featurePrefix := "FEATURE_"
-	if features.CBNT != nil && features.CBNT.Enabled {
+	if features.CBNT.Enabled {
 		hwFeaturesMap[featurePrefix+constants.Cbnt] = strconv.FormatBool(features.CBNT.Enabled)
 		hwFeaturesMap["FEATURE_cbntProfile"] = features.CBNT.Meta.Profile
 	}
-	if features.SUEFI != nil && features.SUEFI.Enabled {
```

Comment:
Very Important to mark these places and use supported flag instead of "!= nil" check, better to add todo

Reply:
Added todo comment.